.helm/starter/templates/awx-deploy.yaml

```
@@ -6,6 +6,10 @@ metadata:
   name: {{ .name }}
   namespace: {{ $.Release.Namespace }}
 spec:
+{{- /* Provide custom persistent volumes configs if enabled */}}
+{{- include "spec.storageClassNames" $ }}
+{{-/* Provide custom secrets configs if enabled */}}
```

Looks like you need a space here.

```suggestion
{{- /* Provide custom secrets configs if enabled */}}
```